jax/_src/pjit.py

```diff
@@ -1592,6 +1593,56 @@ def check_aval_layout_compatibility(
 pjit_p.multiple_results = True
 pjit_p.skip_canonicalization = True
 
+def _is_high(jaxpr, **_) -> bool:
+  return jaxpr.jaxpr.is_high
+pjit_p.is_high = _is_high  # type: ignore
+
+def _lower(*hi_args, jaxpr, out_shardings, out_layouts, **params):
```

Comment:
nit: can you call this _lower_to_lojax because we already have a _lower which means the AOT lower

Reply:
Good nit!

Reply:
Done.